### ISSUE TITLE
Actual Frame Rate is twice the FrameRate setting value under Configuration->Video

### ISSUE THREAD

**Issue by compmas2** [EliasKotlyar/Xiaomi-Dafang-Hacks#1467]
I am using a Wyzecam Pan with Defang Hacks commit 34a96ba from master.
Before I updated recently, I was running a version from Dec. 2019 and this issue was not present.

### Description

Under the Configuration->Video menu, I set the "FrameRate: number of image(s)" to 15 per 1 second.  I am using Zoneminder to watch for motion and it is reporting around 30 fps all the time, so the FrameRate setting is not working.

When running a version of DeFang Hacks from Dec. 2019, I used the same 15 per 1 second setting for FrameRate and my Zoneminder server was picking up 15 fps correctly.  No settings in Zoneminder where changed and only the Defang Hacks firmware update was made.  The bug seems to have been introduced between December 2019 and now.

It appears that the actual frame rate produced is twice the FrameRate setting.  This was verified using other FrameRate settings.  For example a FrameRate of 10 per 1 second results in Zoneminder seeing 20 fps.

### What did you do to debug the issue

Using the same version of Zoneminder (version 1.34.15), I went from a release in December 2019 where zoneminder was reporting 15 fps coming from the camera stream to the current stable release (commit 34a96ba) where zoneminder is reporting around 30 fps.  I have tried setting other FrameRate settings like the following, but I keep getting twice the frames per second in zoneminder compared to the FrameRate value.

It appears that the actual frame rate produced is twice the FrameRate setting.  This was verified using other FrameRate settings.  For example a FrameRate of 10 per 1 second results in Zoneminder seeing 20 fps.

Additionally Zoneminder is setup with no frame rates defines since this could cause issues with obtaining all the frames being sent.  Instead it takes them all as they are produced by the camera.

### Evidence

Here is where I am setting the FrameRate to 15 per 1 seconds:
![image](https://user-images.githubusercontent.com/15125036/83361328-c6b85580-a34d-11ea-9cbd-d455e90ab788.png)

Here is the associated Zoneminder output showing the video at around 30 fps:
![image](https://user-images.githubusercontent.com/15125036/83361356-febf9880-a34d-11ea-887b-98483eafa5a0.png)

Here is where I am setting the FrameRate to 10 per 1 seconds:
![image](https://user-images.githubusercontent.com/15125036/83361410-61b12f80-a34e-11ea-8ec8-3f893ac399f5.png)

Here is the associated Zoneminder output showing the video at around 20 fsp:
![image](https://user-images.githubusercontent.com/15125036/83361432-90c7a100-a34e-11ea-9638-99ee2ac19a80.png)

**Comment by nik0** [comment]
Did you try the latest bêta branch version ?

**Comment by compmas2** [comment]
I just tested the latest beta branch and the same issue is occurring.  I tested both the 10 and 15 per 1 second values and they resulted in 20 and 30 fps respectively.

**Comment by compmas2** [comment]
I am unable to update for some reason.  I went to the update page on the camera web interface and it took several minutes for the update page to populate.  After that it said I was already up to date. Since I have not performed an update for 2 weeks, I forced a beta branch update (delete version file and update option).  During the update I got an error "Error starting update process".  At this point I am unable to update to the newest beta.

Would switching back to Stable and then switching back to beta work?
